base/domain-zul.yml

```diff
@@ -0,0 +1,376 @@
+          Phendula ngelinye lamagama eligqamile ukuze uqhubeke
+
+          MUKELA✅ -_ukwamukela imigomo nemibandela_
+          OKWENGEZIWE 📄 -_ukufunda imigomo nemibandela_
+
+          (noma Phendula ngo-UHLU 📌 _ukubuyela kuhlu lokuqala_)
+
+  utter_more_terms:
+      - text: |
+          Uqinisekisa ukuthi igunya elakho ekunakekelweni nokwelashwa kwakho. Le nsizakalo inikezela ngolwazi kuphela.
+
+          Lensizakalo ayiyona indawo yokwelulekwa ngokwezempilo/ukuzixilonga/ukwelashwa. Thola izeluleko zomhlinzeki wezempilo ofanele mayelana nesimo sakho sezokwelashwa/ ukunakekelwa, ikakhulukazi uma uba nezimpawu ezinzima.
+
+          Uqinisekisa ukuthi akufanele unganaki/ ubambezele ukufuna izeluleko zezokwelashwa mayelana nokwelashwa/ukunakekelwa ngenxa yale nsiza. Thembela kulwazi ngokuqapha.
+
+          Phendula ngelinye lamagama *ELIGQAMILE* ukuze uqhubeke
+          *MUKELA* ✅ ukwamukela imigomo nemibandela
+          (noma Phendula 📌 *UHLU* ukubuyela kuhlu lokuqala)
+
+  utter_ask_province:
+      - text: |
+          Ngabe uhlala esifundazweni siphi manje?
+
+          Phendula:
+          *1.* EASTERN CAPE
+          *2.* FREE STATE
+          *3.* GAUTENG
+          *4.* KWAZULU NATAL
+          *5.* LIMPOPO
+          *6.* MPUMALANGA
+          *7.* NORTH WEST
+          *8.* NORTHERN CAPE
+          *9.* WESTERN CAPE
+
+  utter_incorrect_selection:
+    - text: |
+        Lensizakalo isebenza kahle uma usebenzisa izinketho ezinikezwe *NGOKUGQAMILE*.
+        📌Phendula ngo *UHLU* ukubuyela kuhlu lokuqala
+
+  utter_incorrect_location:
+    - text: |
+        If you have typed your address incorrectly, please try again. If you are unable to provide your address, please TYPE the name of your Suburb, Township, Town or Village (or nearest)
+
+  utter_ask_age:
+    - text: |
+        Uneminyaka emingakhi?
+
+        *1.* <18
+        *2.* 18-39
+        *3.* 40-65
+        *4.*> 65
+
+  utter_ask_gender:
+    - text: |
+        Sicela usinikeze ubulili obuchaza wena:
+
+        Phendula
+        *1.* NGOWESILISA
+        *2.* NGOWESIMAME
+        *3.* OKUNYE
+        *4.* NCAMELA UKUNGASHO
+
+  utter_ask_location:
+    - text: |
+        Sidinga ukwazi indawo okuyo.
+
+        Sicela wabelane ngendawo okuyo 📍 usebenzisa i-WhatsApp ngokulandela lezi zinyathelo:
+        - Cindezela isithonjana sokunamathisela (+ noma 📎) phansi kwe-WhatsApp.
+        - Khetha indawo
+        - Lokhu kuzoletha imephu ngephini elibonisa indawo okuyo manje. Cindezela *Send your location* uma ngabe leli ikheli lakho.
+        - Ungase futhi ukhethe kwezinye izindawo eziseduze uma ucabanga ukuthi i-GPS iyaphazama.
+
+        Noma BHALA igama lesigatshana sakho, iLokishi, Idolobha noma Idolobhana (noma eliseduzane)?
+
+  utter_ask_location_confirm:
+    - text: |
+        ⬛⬛⬛⬜⬜⬜⬜⬜⬜⬜⬜
+
+        Sisebenzisa izinsiza Zendawo zeGoogle ukuqinisekisa amakheli. Sicela uqinisekise ukuthi ikheli elingezansi lilungile ngokwemininingwane osinike yona:
+        {location}
```

Comment:
The location should be surrounded by the `----------------------------` characters